### PR TITLE
Add functions to mount disk for windows guest

### PR DESCRIPTION
**Pull request by cheliu**
Use wmic to get the drive letters assigned
Add get_windows_free_drive_letter() and mount_windows_disk

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -2339,12 +2339,54 @@ def get_windows_drive_letters(session):
...
+    partition_size = normalize_data_size(size, "M")
+    partition_size = str(int(partition_size[:-2]) - 2) + "M"
+
+    if not mountpoint:
+        mountpoint = get_windows_free_drive_letter(session)
+
+    if format_guest_disk(session, did, mountpoint, partition_size, fstype, "windows"):
+        return True
+
+    raise error.TestFail("Formatting guest disk failed.")
```
Comment (xutian) [inline virttest/utils_misc.py]: what's this line do ?

Comment (cheliu) [inline virttest/utils_misc.py]: raise a error if format_guest_disk() failed

**Comment by cheliu** [comment]
ID: 1169678

File: virttest/utils_misc.py [hunk 2]
```diff
@@ -2339,12 +2339,54 @@ def get_windows_drive_letters(session):
...
+
+    :param session: session object to guest
+    :return: a free drive letter
+    """
+    assigned_drive_letters = get_windows_drive_letters(session)
+    all_drive_letters = list(string.ascii_uppercase)
+    drive_letters_free = list(set(all_drive_letters) - set(assigned_drive_letters))
+
+    try:
+        new_drive_letter = drive_letters_free.pop()
```
Comment (xutian) [inline virttest/utils_misc.py]: return drive_letters_free.pop()

**Comment by ypu** [comment]
close this as cheliu is not working on this currently.

**Pull request closed without merging**